sample/Assets/Scripts/SelectAuthMethodScript.cs

```diff
@@ -12,17 +12,31 @@ public class SelectAuthMethodScript : MonoBehaviour
     [SerializeField] private Text Output;
     [SerializeField] private Button UseDeviceCodeAuthButton;
     [SerializeField] private Button UsePKCEButton;
+    string redirectUri = null;

```

Comment:
Redirect URI needs to be set for the PKCE flow.
Device code flow does not need redirectURI.

Also can you make this private too

Reply:
redirectUri was set to null for PKCE for other platforms, what it should be?